### ISSUE TITLE
WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.event__title.listing__item__title)

### ISSUE THREAD

**Issue by aarongustafson** [aarongustafson/aaron-gustafson.com#110]
#### Issue
`Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[My Speaking Engagements (Past & Present) :: Aaron Gustafson](https://www.aaron-gustafson.com/speaking-engagements/)
#### Element path
.event--future > .event__title.listing__item__title
#### Snippet
`<h3 class="event__title listing__item__title">National Disability Rights Network’s Annual Conference</h3>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.94 between the foreground and shadow color (foreground color: #fffcf4, text-shadow color: #38a49a, font size: 27.0pt (36px), font weight: bold). Expected contrast ratio of 3:1
#### Environment
Microsoft Edge version 138.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
